framework/src/Volo.Abp.Ddd.Application/Volo/Abp/Application/Services/CrudAppServiceBase.cs

```diff
@@ -154,9 +154,14 @@ protected virtual void MapToEntity(TUpdateInput updateInput, TEntity entity)
         
         protected virtual void TryToSetTenantId(TEntity entity)
         {
+            if (entity is IMultiTenant tenantEntity && tenantEntity.TenantId.HasValue)
```

Comment:
If entity is not multi-tenant, this if doesn't work. Can you implement this based on my comment: https://github.com/abpframework/abp/pull/1136#issuecomment-493872615

Reply:
`return entity.GetType().GetProperty(nameof(IMultiTenant.TenantId)) != null` like this ?

Reply:
I mean no need to check tenantEntity.TenantId.HasValue at all.
Just create a HasTenantIdProperty method as explained in https://github.com/abpframework/abp/pull/1136#issuecomment-493872615
Thanks.

Reply:
I hope I understand correctly :) otherwise im closing this pr.